purecharity-wp-sponsorships/purecharity-wp-sponsorships.php

```diff
@@ -89,4 +89,32 @@ function ss_force_template() {
 	$options = get_option( 'purecharity_sponsorships_settings' );
   include(get_template_directory() . '/' . $options['single_view_template']);
   exit;
+}
+
+/*
+ * Plugin updater using GitHub
+ *
+ * Auto Updates through GitHub
+ *
+ * @since   1.1.2
+ */
+add_action( 'init', 'purecharity_wp_sponsorships_updater' );
+function purecharity_wp_sponsorships_updater() {
+  if ( is_admin() ) { 
+    $sp_config = array(
+      'slug' => plugin_basename( __FILE__ ),
+      'proper_folder_name' => 'purecharity-wp-sponsorships',
+      'api_url' => 'https://api.github.com/repos/purecharity/pure-sponsorships',
+      'raw_url' => 'https://raw.github.com/purecharity/pure-sponsorships/master',
+      'raw_url' => 'https://raw.githubusercontent.com/purecharity/pure-sponsorships/master/purecharity-wp-sponsorships/',
```

Comment:
I'm not sure if this is intentional but there are two entries for `raw_url` here